examples/api-routes-graphql/README.md

```diff
@@ -1,6 +1,24 @@
 # API routes with GraphQL server
 
-Next.js ships with [API routes](https://nextjs.org/docs/api-routes/introduction), which provide an easy solution to build your own `API`. This example shows their usage alongside [apollo-server-micro](https://github.com/apollographql/apollo-server/tree/main/packages/apollo-server-micro) to provide simple GraphQL server consumed by Next.js app.
+Next.js ships with [API routes](https://nextjs.org/docs/api-routes/introduction), which provide an easy solution to build your own `API`.
```

Comment:
Is there a deployed example of this we can see?

Reply:
Sure, you'll find this branch deployed on Vercel here: https://api-routes-graphql-yoga.vercel.app/api/graphql?query=%7B%0A++greetings%0A%7D